src/main/java/cloud/orbit/actors/extensions/metrics/dropwizard/OrbitActorExtension.java

```diff
@@ -137,40 +141,53 @@ public void onRead(HandlerContext ctx, Object message)
     {
         //actor counter per type and node
         String actorCounterKey = getActorCounterMetricsKey(RemoteReference.getInterfaceClass(actor));
-        Counter counter = MetricsManager.getInstance().getRegistry().counter(actorCounterKey);
-        counter.dec();
+        MetricsManager.getInstance().getRegistry().counter(actorCounterKey).dec();
 
-        //actor lifespan
-        String startTimeKey = getActorStartTimeKey(actor);
+        //actor deactivation
+        String actorDeactivationTimerKey = getDeactorActivationTimerKey(actor);
+        Timer.Context deactivationTimer = actorDeactivationTimers.remove(actorDeactivationTimerKey);
+        if (null != deactivationTimer)
+        {
+            deactivationTimer.stop();
+        }
 
-        Long startTime = actorStartTimes.get(startTimeKey);
-        if (null != startTime)
+        //Actor lifespan
+        String actorLifespanTimerKey = getActorLifespanTimerKey(actor);
```

Comment:
Move this to preDeactivation based on conversation with Joe. We want activation to go from preActivation to postActivation, lifespan to go from postActivation to preDeactivation, and deactivation to go from preDeactivation to postDeactivation.